devicetypes/smartthings/zigbee-window-shade-battery.src/zigbee-window-shade-battery.groovy

```diff
@@ -67,9 +67,15 @@ metadata {
 		standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 1) {
 			state "default", label:"", action:"refresh.refresh", icon:"st.secondary.refresh"
 		}
+		valueTile("shadeLevel", "device.level", width: 4, height: 1) {
```

Comment:
While the tiles are not used any more, for the sake of cleanliness, let's please not add this and below to the PR.